cli/flox/src/commands/init/python.rs

```diff
@@ -795,4 +829,62 @@ mod tests {
             poetry_version: "1.7.1".to_string(),
         });
     }
+
+    /// Requirements::detect should return None if no requirements.txt is found
+    #[test]
+    #[serial]
```

Comment:
nonblocking: these only have to be `serial` because of the `get_default_package_if_compatible` call. If you split out a helper from `detect` these unit tests could be a bit more targeted and we could get rid of the `serial`